### PR TITLE
Store schema locally

### PR DESCRIPTION
**Pull request by Carreau**
See #108

### CODE REVIEW

File: .github/workflows/sync_schema.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: CI
```
Comment (Carreau) [inline .github/workflows/sync_schema.yml:1]:
```suggestion
name: Schema Sync
```

**Comment by tlambert03** [comment]
Curious what @nclack and @czaki think here. 

#83 was going to do this and then specifically closed after discussion in meeting

I could be swayed, but just wanna hear thoughts again

**Comment by tlambert03** [comment]
Minimally, @Carreau, I think we'll  need to be careful when checking out the schema elsewhere (eg for docs), to specifically checkout the latest release, and not main.

**Comment by Czaki** [comment]
But why store schema locally instead of generating it dynamically?

**Comment by nclack** [comment]
The main problem here is that `test_render_docs` requires an internet connection. I think it's reasonable to fall back to using the local schema for the purposes of testing when urlopen fails.  I think this just needs a try around the `urlopen`-block in `render.py` (see #112).

To me `schema.json` feels like a kind of documentation artifact so I'd rather not commit it back to the repo.

**Comment by tlambert03** [comment]
Yeah; since we've already been through this discussion and closed a related PR to this effect, I'm going to close this.  We can discuss in group meeting anything we may have overlooked.  I agree with falling back to rebuilding the schema if there's no Internet.

**Pull request closed without merging**